### ISSUE TITLE
Incorrectly prefixed composer/InstalledVersion.php array_merge call_user_func_array

### ISSUE THREAD

**Issue by kkmuffme** [humbug/php-scoper#1033]
## Bug report

| Question    | Answer
| ------------| ---------------
| PHP-Scoper version | 0.18.11
| PHP version | 7.4-8.3
| Platform with version | CentOS 9

Composer 2.7.6

InstalledVersions.php incorrectly prefixed `array_merge` with the prefix:

```
return array_keys(array_flip(\call_user_func_array('array_merge', $packages)));
```

to

```
return \array_keys(\array_flip(\call_user_func_array('My\\Foo\\array_merge', $packages)));
```

This can be fixed with a custom patcher easily, however since this affects all php-scoper users, I think it makes sense to fix it in scoper directly to NOT prefix this

This worked correctly before 
https://github.com/humbug/php-scoper/pull/889 (as a side effect of https://github.com/humbug/php-scoper/pull/864) was implemented.

**Comment by theofidry** [comment]
I could indeed reproduce the issue on 0.18.11, it is however fixed on master. I hope to release it within the week or next week.

**Comment by theofidry** [comment]
Released in 0.18.12
